src/Uno.UI.Runtime.Skia.Gtk/Rendering/GLRenderSurfaceBase.cs

```diff
@@ -83,6 +83,12 @@ private void GLRenderSurface_Realized(object? sender, EventArgs e)
 
 		private void UnoGLDrawingArea_Render(object o, RenderArgs args)
 		{
+			if (_host.RootElement is { } rootElement &&
+				(rootElement.IsArrangeDirtyOrArrangeDirtyPath || rootElement.IsMeasureDirtyOrMeasureDirtyPath))
+			{
+				rootElement.UpdateLayout();
+			}
+
```

Comment:
@Youssef1313 Shouldn't we do this in a more general location? This may happen for other skia targets.

Reply:
Somehow, not happening on WPF, macOS, X11.